opm/parser/eclipse/Units/UnitSystem.cpp

```diff
@@ -57,6 +57,9 @@ namespace {
         1 / ( Metric::ReservoirVolume / Metric::Time ),
         1 / Metric::Transmissibility,
         1 / Metric::Mass,
+        1, /* gas-oil ratio */
+        1, /* oil-gas ratio */
```

Comment:
Yes, oil-gas is for a gas condensate field/reservoir condensate yield
normally called CGR

Regards

David Baxendale
david.baxendale@gmail.com

On Tue, Jul 19, 2016 at 9:27 PM, Joakim Hove <notifications@github.com>
wrote:

> In opm/parser/eclipse/Units/UnitSystem.cpp
> <https://github.com/OPM/opm-parser/pull/885#discussion_r71337149>:
>
> > @@ -57,6 +57,9 @@ namespace {
> >          1 / ( Metric::ReservoirVolume / Metric::Time ),
> >          1 / Metric::Transmissibility,
> >          1 / Metric::Mass,
> > +        1, /* gas-oil ratio */
> > +        1, /* oil-gas ratio */
>
> Do we need both gas-oil and oil-gas?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OPM/opm-parser/pull/885/files/afbe2be7d7f2733224820d83c4aaabe742832e87#r71337149>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARIIDxIh-cUdxPRZKPyKMDcc3pB6_bYOks5qXNDAgaJpZM4JPsFs>
> .
>